netbox_slm/models.py

```diff
@@ -113,12 +114,25 @@ class SoftwareLicense(NetBoxModel):
         null=True,
         blank=True
     )
+    license_key = models.CharField(max_length=255, null=True, blank=True)
+    license_count = models.IntegerField(
+        verbose_name="license count",
+        blank=True
+    )
     installation = models.ForeignKey(
         to='netbox_slm.SoftwareProductInstallation',
         on_delete=models.PROTECT,
         null=True,
         blank=True
     )
+    users = models.TextField(
+        verbose_name='users',
+        blank=True
+    )
+    comments = models.TextField(
```

Comment:
How are these additional fields `license_count`, `users`, and `comments` intended to be used?

Reply:
for example a confluence Server License:
license_count: number of licensed Users
users: users currently assigned
comment: users get there license via ldap group synchronization

Reply:
Oh, that's interesting. Perhaps you can also open a new issue (feature request) in this repository and we can work out the details. I'll also be adding an extra storage location URL field to point to the license key (stored externally)